packages/react-ui-components/src/components/SchemaForm/SchemaForm.module.scss

```diff
@@ -0,0 +1,82 @@
+@import "node_modules/@kelvininc/ui-components/dist/assets/styles/globals.scss";
+
+.form-container {
+	--schema-form-x-padding: 0;
```

Comment:
You use sass variables instead of css custom properties, syntax is cleaner.

Reply:
I use a css var because I need set these vars from outside

Reply:
Understood 👍